app/views/viewall/alumni.html.erb

```diff
@@ -1,4 +1,5 @@
 <h2 align="center">Complete Alumni Database</h2>
+<h4 align="center">Total Count: &nbsp; <label class="label label-info"> <%= @allAlums.count %> </label></h4>
```

Comment:
The label is not necessary here. The UI of the info label is slightly screwed up, I can't seem to make out the numbers clearly.